[CI][dashboard] Collect PT2 cpu perf nightly

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #131369

Summary: Add a workflow similar to inductor-perf-test-nightly.yml but use x86 metal instances for perf measurement. The data processing and dashboard update will come next.

desertfire committed:
[CI][dashboard] Collect PT2 cpu perf nightly

Summary: Add a workflow similar to inductor-perf-test-nightly.yml but use x86 metal instances for perf measurement. The data processing and dashboard update is TBD.

[ghstack-poisoned]

desertfire committed:
Update on "[CI][dashboard] Collect PT2 cpu perf nightly"


Summary: Add a workflow similar to inductor-perf-test-nightly.yml but use x86 metal instances for perf measurement. The data processing and dashboard update will come next.

[ghstack-poisoned]

huydhn reviewed (approved):
LGTM!  We will need https://github.com/pytorch/test-infra/pull/5491 to go with this to surface the CPU benchmark results on the dashboard under a new device.

desertfire committed:
Update on "[CI][dashboard] Collect PT2 cpu perf nightly"


Summary: Add a workflow similar to inductor-perf-test-nightly.yml but use x86 metal instances for perf measurement. The data processing and dashboard update will come next.

[ghstack-poisoned]

desertfire commented:
@pytorchbot merge -f "affects dashboard CI only"

atalman commented:
@pytorchmergebot revert -c nosignal -m "Sorry need to revert looks like , please run ciflow/inductor looks like this caused failure in https://github.com/pytorch/pytorch/actions/runs/10058412015/job/27802257096"

atalman commented:
@pytorchmergebot revert -c nosignal -m "Sorry need to revert looks like , please run ciflow/inductor looks like this caused failure in [pytorch/pytorch/actions/runs/10058412015/job/27802257096](https://github.com/pytorch/pytorch/actions/runs/10058412015/job/27802257096)"
